field.js

```diff
@@ -0,0 +1,21 @@
+class Field {
+  td = document.createElement('td');
+
```

Comment:
field should contain state property if hidden or shown, it should also contain information if it contains the bomb or how many bombs are arround it. Base on those information it should be rendered differentyl.